packages/app/Dockerfile

```diff
@@ -34,4 +34,4 @@ ENV PORT $PORT
 USER node
 WORKDIR /usr/src/app/packages/app/dist
 
-CMD http-server -p $PORT
\ No newline at end of file
+CMD http-server -p $PORT -c-1 --proxy="http://127.0.0.1:$PORT/index.html?"
```

Comment:
`--proxy` - setting to properly work with SPA and route nested paths requests to `/index.html` otherwise page reload on nested pages (e.g. `http://localhost:3010/block/1`) gives 404.
`-c-1` - disable http server cache.